tests/conftest.py

```diff
@@ -414,3 +446,46 @@ def fixture_aggr_surfs_mean(fmurun_w_casemetadata, rmsglobalconfig, regsurf):
     os.chdir(origfolder)
 
     return (aggregated["mean"], metas)
+
+
+# ======================================================================================
+# Utilities
+# ======================================================================================
+
+
+def _parse_json(schema_path):
+    """Parse the schema, return JSON"""
+    with open(schema_path) as stream:
+        data = json.load(stream)
+
+    return data
+
+
+def _parse_yaml(yaml_path):
+    """Parse the filename as json, return data"""
+    with open(yaml_path, "r") as stream:
+        data = yaml.safe_load(stream)
+
+    data = _isoformat_all_datetimes(data)
+
+    return data
+
+
+def _isoformat_all_datetimes(data):
+    """Recursive function to isoformat all datetimes in a dictionary"""
+
+    if isinstance(data, list):
+        data = [_isoformat_all_datetimes(i) for i in data]
+        return data
+
+    if isinstance(data, dict):
+        for key in data:
+            data[key] = _isoformat_all_datetimes(data[key])
+
+    if isinstance(data, datetime.datetime):
+        return data.isoformat()
+
+    if isinstance(data, datetime.date):
+        return data.isoformat()
```

Comment:
These can be merget with:
```
if isinstance(data, (datetime.datetime, datetime.date)):
    return data.isoformat()

```
Perhaps I would have use `indate` instead of `data` in the whole function